.github/pull_request_template.md

```diff
@@ -37,8 +34,9 @@ Describe how the changes were tested and how reviewers can test them too:
 
 Before submitting the PR, ensure the following:
 
-- [ ] 🔍 PR title is clear and descriptive
-- [ ] 📝 For internal contributors: If applicable, include the JIRA ticket number (e.g., ITEP-123456) in the PR **title**. Do **not** include full URLs
+- [ ] 🔍 PR title is clear and meaningful
+- [ ] ✍️ PR description clearly explains the changes and their reason
+- [ ] 📝 I have linked the PR to the corresponding GitHub Issues, if any
 - [ ] 💬 I have commented my code, especially in hard-to-understand areas
 - [ ] 📄 I have made corresponding changes to the documentation
 - [ ] ✅ I have added tests that prove my fix is effective or my feature works
```

Comment:
I think this checklist is a bit too much to have on every PR, and IMO it clutters the PR overview. What do you think of removing it?